core/main.go

```diff
@@ -265,7 +265,7 @@ func (t *Textile) Start() error {
 
 	t.online = make(chan struct{})
 	t.updates = make(chan Update, 10)
-	t.threadUpdates = make(chan ThreadUpdate, 10)
+	// t.threadUpdates = make(chan ThreadUpdate, 10)
```

Comment:
can be removed, left for review... should be gone. broadcaster is lazily instantiated, so don't need this

Reply:
nice! kill it